doc/edge-metrics-v2.md

```diff
@@ -0,0 +1,43 @@
+# IoT Edge Metrics
+
+## IoT Edge Hub
```

Comment:
Not: Everywhere else in the repo, the EdgeHub is referred to as EdgeHub and not IoT Edge Hub.